webapp/src/Service/ImportExportService.php

```diff
@@ -543,10 +545,14 @@ public function importJson(string $type, UploadedFile $file, ?string &$message =
         $content = file_get_contents($file->getRealPath());
         try {
             $data = $this->dj->jsonDecode($content);
-        }
-        catch (\JsonException $e) {
-            $message = "File contents is not valid JSON: " . $e->getMessage();
-            return -1;
+        } catch (\JsonException $e) {
+            // Check if we can parse it as YAML
+            try {
+                $data = Yaml::parse($content, Yaml::PARSE_DATETIME);
+            } catch (ParseException $e) {
+                $message = "File contents is not valid JSON or YAML: " . $e->getMessage();
+                return -1;
```

Comment:
We only need to return numeric exit codes in Commands right?

Reply:
This is based on other imports here. Id like to be consistent so let’s not change it now